### DISCUSSION TITLE
[2023 HOMEWORK TURN-IN] Post links to "turn in" your homework here.

### DISCUSSION THREAD

**Discussion by williamgilpin** [williamgilpin/cphy#20] [archived]
Please post a link to your completed assignments in this thread. The schedule and syllabus are [here](https://docs.google.com/document/d/1URJmdpTVG8E2bLLu5xAHctICb6krbZ0fC0hO2i2xEXY/edit?usp=sharing)

Feel free to either: Link to the assignment in a fork of the full course repo; link to just the completed assignment (preferably still host it on your GitHub); or upload a `.ipynb` or `.py` file using [GitHub gists](https://gist.github.com/).

+ To maximize the value of this course to you, I'd recommend attempting the problems on your own first before consulting my solutions.

+ Please put your name in your post somewhere, so I know who's who

**Comment by ycaophysics** [comment]
Attached is the link to my Github gists:
Thank you 
Yuxuan Cao
https://gist.github.com/ycaophysics/5e443353606d4ac52d7c7b479aa40fca

**Comment by timothyara** [comment]
https://github.com/timothyara/test_repo/blob/4df8a5f1316eeab2d8471c8f4438b9d09176e50b/Sandpile%26Percolation.ipynb

^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^Link to code ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

-Timothy

**Reply by williamgilpin** [reply]
I'm getting a 404, is your repo private? Check in Incognito

<img width="1306" alt="Screen Shot 2023-09-13 at 11 25 49 AM" src="https://github.com/williamgilpin/cphy/assets/8154246/db4c22ad-ee4f-4f32-b70b-4fef015f2171">

**Comment by gpsehr** [comment]
Attached is the link to my code:

https://github.com/gpsehr/CompPhysics/blob/main/cellular_automata_complexity.ipynb

Thank you,
Gregory Sehr

**Comment by JImmyHCY** [comment]
My HW1 is attached, Thanks,
 
https://github.com/JImmyHCY/CPHY/blob/af58111523daed00b2d1886d3f98c9ed650881cb/HW1-Ching%20Yu%20Huang.ipynb

Ching-Yu Huang (ch52423)

**Comment by clobrien-ut** [comment]
Chloe O'Brien HW1
https://gist.github.com/clobrien-ut/9b690d079c287a1f1c6bbac92aa29072

**Comment by ccyehintx** [comment]
Hi,
The link to my HW1 is attached below:
https://gist.github.com/ccyehintx/f48b40863982ca0e8b91fdb865a0d193
Thank you

p.s. I have tried to attempt the river model but the results did not agree with the publication. My work on this is also in the link
Chieh-Chih (George) Yeh

**Reply by williamgilpin** [reply]
thanks for attempting it, i'll take a look at your code and the PRL paper and see if I can reverse engineer it. In a future year, I'd love to replace the percolation problem with this river CA, which from the PRL paper looks really unique

**Comment by abbyperryman** [comment]
Abigail Perryman (asp3337):
https://github.com/abbyperryman/cphy/blob/main/actualHW1.py

**Comment by shoshanareich** [comment]
Shoshi Reich 
sar5447

https://github.com/shoshanareich/computational_physics/blob/main/cellular_automata_complexity.ipynb

**Comment by laurenlarson** [comment]
Lauren Larson 
lml3638

https://github.com/laurenlarson/cphy/blob/6850a592ad966f33afa241d4d2e4f084038af0c7/hw/cellular_automata_complexity.ipynb

**Comment by bhollo0329** [comment]
Bryce Holloway
bh28929
https://github.com/bhollo0329/CompPhys2023/blob/main/HW1.ipynb

**Comment by sepstein22** [comment]
Sophia Epstein
see873
[https://github.com/sepstein22/cphy_23/blob/main/hw/cellular_automata_complexity.ipynb](url)

**Reply by abao1999** [reply]
Hi Sophia, I got a 404 error when I tried clicking on your link. Maybe it's a private repo? Let me know if any questions come up! Just wanted to check in

**Comment by DanUrban000** [comment]
Daniel Urbanski
Deu223
https://github.com/DanUrban000/Computational_Physics/blob/main/Urbanski_HW_1.ipynb

**Reply by abao1999** [reply]
Hi Daniel, I got a 404 error when clicking on your link. Maybe it's a private repo? Lmk if anything comes up or if I can be of assitance :)

**Reply by abao1999** [reply]
Great! I see it now

**Comment by ravikoka** [comment]
Ravi Koka (rkk582)

Here's the link to my first homework: https://github.com/ravikoka/comp_phy/blob/main/hw1.ipynb

**Comment by MercuryLYC** [comment]
Yuchen Lin(yl38994)

https://github.com/MercuryLYC/test_repo/blob/main/HW1%20yl38994.ipynb

**Comment by Art-of-FUgue** [comment]
Hongming Zhang
hz6835
https://github.com/Art-of-FUgue/HW/blob/main/cellular_automata_complexity.ipynb

**Comment by ConcreteWang** [comment]
Tong Wang
tw28927
https://github.com/ConcreteWang/Test1/blob/main/cellular_automata_complexity.ipynb

**Comment by klanderson42** [comment]
Hi,
Kody Anderson
kla2648

My homework 1 is here: https://github.com/klanderson42/cphy-ocaml/blob/main/hw/cellular_automata_complexity.ipynb

The repo is a mess right now, also callled OCaml - I wanted to implement in OCaml but was having install issues. Will revisit later. The OCaml build system (dune) generates a lot of files, so they're here for now.

**Reply by klanderson42** [reply]
Realized I was still importing the solution for testing. The sandpile required a few changes, but the percolation needed a pretty big rewrite.

**Comment by shvnmttl** [comment]
[HW1 Name: Shivan Mittal, UTEID: sm75447](https://github.com/shvnmttl/comphy/blob/42873c8190a5420251f9dcc2c44ddc8b11133697/hw/hw1.ipynb)

**Comment by savpinedo28** [comment]
Savannah Pinedo
sap4423
https://github.com/savpinedo28/cphy/blob/d13b95fcd443aba024961d8d61ff55f33a59ee89/cellular_automata_complexity_pinedo.ipynb

**Comment by DarkQuark26** [comment]
Patrick Devaney
UT EID: pd9552
Link to HW 1: https://github.com/DarkQuark26/cphy/blob/main/hw/cellular_automata_complexity.ipynb

**Comment by joseywright** [comment]
Josephina/Josey Wright
Link to my code: https://github.com/joseywright/computationalHomework/blob/main/Homework1.ipynb

**Comment by ehansen99** [comment]
[Erik Hansen, UT EID: eh29649](https://github.com/ehansen99/cphy/blob/main/hw/hansen_hw1.ipynb)

I had a bit of trouble determining the time and memory scaling of my percolation model so I might appreciate feedback about that.

**Reply by williamgilpin** [reply]
the scaling might be hard to determine empirically, it should be somewhere between $\mathcal{O}(N)$ and $\mathcal{O}(N^2)$, where $N$ is the lattice linear size. Usually it's closer to $\mathcal{O}(N^2)$ in the dilute limit (since we end up filling most of the sites), but in the weakly percolating regime it's closer to $\mathcal{O}(N)$ because the pathlength of the thread connecting top to bottom scales linearly. In the clogged limit it's basically constant because the simulation dies in the first few rows.

nice plots btw

**Comment by Potatoasad** [comment]
Asad Hussain
EID: ah54594
Link to my code: [Here it is](https://github.com/Potatoasad/cphy/blob/asad_solutions/hw/cellular_automata_complexity.ipynb). It's on my own branch on my own fork, so that I don't mess with my main.

**Reply by williamgilpin** [reply]
nice use of github & tqdm

**Comment by bluejelli** [comment]
[Homework 1: Anish Pandya; ap53535](https://gist.github.com/bluejelli/becdd78b583293f48e1fd862e3fa3c9d)

**Comment by nlindsellTexas** [comment]
https://github.com/nlindsellTexas/CompPhys

**Comment by BryanD-UT** [comment]
https://utexas.box.com/s/vnr2uuo25xsdwvow3q9bgbymhg60h73e

**Comment by htphy** [comment]
Hanting Chen hc28747[
](https://github.austin.utexas.edu/hc28747/test_repo/blob/main/hw1/cellular_automata_complexity.ipynb)

**Reply by htphy** [reply]
https://github.austin.utexas.edu/hc28747/test_repo/blob/main/hw1/cellular_automata_complexity.ipynb

**Comment by aditi-pujar** [comment]
Aditi Pujar, ap59797 - [HW1](https://github.com/aditi-pujar/cphy/blob/main/hw/cellular_automata_complexity.ipynb)

**Comment by DanUrban000** [comment]
Hi, mine should be public now. Thank you.

**Comment by sepstein22** [comment]
Sophia Epstein, see873: [HW2](https://github.com/sepstein22/cphy_23/blob/cc94d8b39d386cce9f6b79012022bd68bec5d849/hw/matrices_unsupervised_learning.ipynb)

**Comment by shoshanareich** [comment]
Shoshi Reich
sar5447

https://github.com/shoshanareich/computational_physics/blob/main/matrices_unsupervised_learning.ipynb

**Comment by timothyara** [comment]
Timothy Araujo
tda739

https://github.com/timothyara/test_repo/blob/435d921437f66b50c6ba61b5330f8e4430f50547/HW%232.ipynb

**Comment by joseywright** [comment]
Homework 2 turn in: https://github.com/joseywright/computationalHomework/blob/main/Homework2.ipynb
eid:jrw5288

**Comment by abbyperryman** [comment]
Homework 2 submission: https://github.com/abbyperryman/cphy/blob/main/hw2.ipynb
eid: asp337

**Comment by bhollo0329** [comment]
Bryce Holloway
bh28929
HW2
https://github.com/bhollo0329/CompPhys2023/blob/main/HW2.ipynb

**Comment by laurenlarson** [comment]
Lauren Larson
eid: lml3638
https://github.com/laurenlarson/cphy/blob/7cbb47e2c7cfa31fa669e781d9a6dd57024fc65d/hw/matrices_unsupervised_learning.ipynb

**Comment by clobrien-ut** [comment]
Chloe O'Brien
alb5374
https://github.com/clobrien-ut/cphy/blob/main/obrien.matrices_unsupervised_learning.ipynb

**Comment by ycaophysics** [comment]
HW2 turn in: https://gist.github.com/ycaophysics/52d70a45c2c3a60eff3da124d081a560
EID yc29393

**Comment by savpinedo28** [comment]
Savannah Pinedo
sap4423
https://github.com/savpinedo28/cphy/blob/main/hw2_pinedo.ipynb

**Comment by DarkQuark26** [comment]
Patrick Devaney
UT EID: pd9552

HW 2
https://github.com/DarkQuark26/cphy/blob/main/patrick_devaney_hw_solutions/matrices_unsupervised_learning.ipynb

**Comment by nlindsellTexas** [comment]
Noah Lindsell
nl22437
See my homework repo here. It has Hw 2 in it 

https://github.com/nlindsellTexas/CompPhys

**Comment by klanderson42** [comment]
Kody Anderson
kla2648

HW 2: https://github.com/klanderson42/cphy-ocaml/blob/main/hw/matrices_unsupervised_learning.ipynb

Let me know if it doesn't open, I was getting ".ipynb is not a valid JSON file" but I think it might be my browser. Also, I couldn't get my PCA to run (after ~20 minutes). I'll keep trying. Most of my answers to conceptual questions are in new markdown cells.

**Reply by williamgilpin** [reply]
I am getting a bug when I try to open on github and VSCode, did the ipynb file get corrupted?

**Reply by klanderson42** [reply]
I think it was. I think it should be fixed now.

**Reply by abao1999** [reply]
I'll take a look at the PCA thing tonight. At first glance, looks like just a small bug

**Comment by Art-of-FUgue** [comment]
Hongming Zhang
hz6835
https://github.com/Art-of-FUgue/HW/blob/main/matrices_unsupervised_learning.ipynb

**Comment by ConcreteWang** [comment]
Tong Wang
tw28927
https://github.com/ConcreteWang/Test1/blob/main/Homework2.ipynb

**Comment by ravikoka** [comment]
Ravi Koka
rkk582
hw 2 link: https://github.com/ravikoka/comp_phy/blob/1fd6220bbd5e39c6170c3552a3bfb83654fde729/hw2_matrices_unsupervised_learning.ipynb

**Comment by MercuryLYC** [comment]
Yuchen Lin
yl38992
https://github.com/MercuryLYC/test_repo/blob/main/HW2_1%20yl38994.ipynb

**Comment by bluejelli** [comment]
Anish Pandya
ap53535
[hw2](https://gist.github.com/bluejelli/16a2ad929a4c35c3a4746de64fe75332)

**Comment by JImmyHCY** [comment]
Ching-Yu Huang
ch52423
https://github.com/JImmyHCY/CPHY/blob/6cac2347378ffe9e560e74664f6d68b12f8be936/HW2.ipynb

**Comment by DanUrban000** [comment]
Daniel Urbanski
deu223
https://github.com/DanUrban000/Computational_Physics/blob/main/Urbanski_HW_2.ipynb

**Comment by htphy** [comment]
Han-Ting Chen
hc28747
https://github.austin.utexas.edu/hc28747/test_repo/blob/main/hw2/matrices_unsupervised_learning.ipynb

**Comment by BryanD-UT** [comment]
Attached is my assignment: https://utexas.box.com/s/xhbebsgnu4uyjgv50kg9d2r0nb9q1npr
Thanks,
Bryan Dinh, bvd249

**Comment by gpsehr** [comment]
Attached is my HW2 submission: https://github.com/gpsehr/CompPhysics/blob/main/matrices_unsupervised_learning.ipynb

Thank you,
Gregory Sehr, gps687

**Comment by shvnmttl** [comment]
I was having issues with pushing my project to git. So, I zipped my HW and attached that here. I will keep trying to get the push to work and put a link in the reply to this post when I have one available. I apologize for the inconvenience. 
Thank you for understanding,
Shivan Mittal sm75447

[hw2.zip](https://github.com/williamgilpin/cphy/files/12812072/hw2.zip)

**Reply by shvnmttl** [reply]
Got it working, [this](https://github.com/shvnmttl/comphy/blob/ca771ad03a22937aca82a278893aaed25d5ffe14/hw/hw2.ipynb) is the link to my homework.

**Reply by abao1999** [reply]
No worries! this looks great

**Comment by ehansen99** [comment]
My homework submission is in my GitHub fork [here](https://github.com/ehansen99/cphy/blob/main/hw/hansen_hw2.ipynb). I did have some difficulty running my PCA code, as it kept killing my kernel when moving to higher Reynolds number, so I might appreciate some feedback on that.

**Reply by abao1999** [reply]
I'll take a look! (prob late at night or over the weekend though : / )

**Reply by williamgilpin** [reply]
if you import my solutions, does it still die? trying to figure out if its code or your hardware.

**Reply by ehansen99** [reply]
Your solution worked for my kernel for Re = 600-1200; I believe mine ran for Re = 300 but none of the others.

**Comment by ccyehintx** [comment]
Hi my homework is attached in this link here:
https://gist.github.com/ccyehintx/adc5157500c537a7b1e14c5d918142bc
Sorry that the wifi was down for the last hour so I couldn't use my laptop as I only have the link in my laptop

Chieh-Chih (George) Yeh (cy7658)

**Reply by abao1999** [reply]
Looks good - if you ever have the wifi issues in the future, just lmk! Happy to accommodate, so no need to be stressed too much : )

**Comment by Potatoasad** [comment]
My homework is attached here:
https://github.com/Potatoasad/cphy/blob/asad_solutions/hw/matrices_unsupervised_learning.ipynb

Asad Hussain
ah54594

**Comment by aditi-pujar** [comment]
Really sorry I couldn't get this done in time, I do hope it may still be considered:
[HW2](https://github.com/aditi-pujar/cphy-fall-23/blob/main/hw/my-solutions/HW2__matrices_unsupervised_learning.ipynb)

Aditi Pujar
ap59797

**Reply by abao1999** [reply]
of course! Don't stress about submitting exactly on time - there's a grace period and we try to be understanding

**Comment by timothyara** [comment]
Timothy Araujo
tda739
HW 3 Link:
https://github.com/timothyara/test_repo/blob/9c60a43380f50143fb7f63dba75eab041be12ba4/HW%233PDE.ipynb

**Reply by williamgilpin** [reply]
looks great, some sort of weird instability in the spectral solver, but i don't see what's causing it. i'll see if the TA(s) can see what's causing it

**Comment by ccyehintx** [comment]
Hi,
Please see attched of my hw3:
https://gist.github.com/ccyehintx/5948255c612f12ef94539b8b816546ac

Thank you
Chieh-Chih (George) Yeh (cy7658)

**Reply by williamgilpin** [reply]
hey i can't see the gray-scott, is it a different gist? william

**Comment by ravikoka** [comment]
Ravi Koka
uteid: rkk582

Link to hw3:
https://github.com/ravikoka/comp_phy/blob/385e4f2bdee58fa89f984af21b2cc31834a421d3/hw3_pde_turing.ipynb

**Comment by gpsehr** [comment]
Attached is my HW3 submission: https://github.com/gpsehr/CompPhysics/blob/main/Sehr_HW3.ipynb

Thank you,
Gregory Sehr, gps687

**Comment by bhollo0329** [comment]
Bryce Holloway
bh28929
HW3: https://github.com/bhollo0329/CompPhys2023/blob/main/HW3.ipynb

**Comment by abbyperryman** [comment]
Abigail Perryman
asp3337
https://github.com/abbyperryman/cphy/blob/main/hw3.ipynb

**Comment by clobrien-ut** [comment]
Chloe O'Brien
alb5374
https://github.com/clobrien-ut/cphy/blob/main/obrien.pde_turing.ipynb

**Comment by sepstein22** [comment]
Sophia Epstein 
see873
https://github.com/sepstein22/cphy_23/blob/e34b889748f1922068ce8323ff1f917b11a55307/hw/pde_turing.ipynb

**Comment by laurenlarson** [comment]
Lauren Larson 
lml3638
https://github.com/laurenlarson/cphy/blob/0af37c9b7f77ad8b7487f381d4db046dc022d3a5/hw/pde_turing.ipynb

**Comment by DanUrban000** [comment]
Daniel Urbanski
deu223
https://github.com/DanUrban000/Computational_Physics/blob/main/Urbanski_HW_3.ipynb

**Comment by MercuryLYC** [comment]
Yuchen Lin
yl38994
https://github.com/MercuryLYC/test_repo/blob/main/HW3%20yl38994.ipynb

**Comment by shoshanareich** [comment]
Shoshi Reich
sar5447
https://github.com/shoshanareich/computational_physics/blob/main/pde_turing.ipynb

**Comment by klanderson42** [comment]
Kody Anderson
kla2648
Here's a link to the gist:
https://gist.github.com/klanderson42/e66242760f6b971695052a8c9b4a5196
The Gray Scott code isn't working, I think it's a problem with the hstack. I'll keep looking.

**Comment by joseywright** [comment]
jrw5288
https://github.com/joseywright/computationalHomework/blob/main/Homework3.ipynb

**Comment by htphy** [comment]
Han-Ting Chen
hc28747
https://github.austin.utexas.edu/hc28747/test_repo/blob/main/hw3/pde_turing.ipynb

**Comment by ycaophysics** [comment]
HW3
Yuxuan Cao 
yc29393
https://gist.github.com/ycaophysics/c77a885f210789d64fc075185639d7e9

**Comment by JImmyHCY** [comment]
HW3
Ching-Yu Huang
ch52423
https://github.com/JImmyHCY/CPHY/blob/ee6ad640952e1dda5f439ac47e236c5f55d840ce/HW3.ipynb

**Comment by Art-of-FUgue** [comment]
HW3
hz6835 
Hongming Zhang
https://github.com/Art-of-FUgue/HW/blob/main/pde_turing.ipynb

**Comment by nlindsellTexas** [comment]
HW3
Noah Lindsell
nl22437
https://github.com/nlindsellTexas/CompPhys

**Comment by BryanD-UT** [comment]
HW3 
Bryan Dinh
bvd249
https://utexas.box.com/s/ht9flf62hlos7oot6gdzwhzpo5gxnf47

**Comment by ehansen99** [comment]
Erik Hansen
eh29649
[https://github.com/ehansen99/cphy/blob/main/hw/hansen_hw3.ipynb](https://github.com/ehansen99/cphy/blob/main/hw/hansen_hw3.ipynb)

**Comment by shvnmttl** [comment]
My homeowrk is attached as a zip file here, I will upload it to my github and attach a link as well. I will post a repy to this post.
For some reason I had a hard time running a solution with low damping and large beta for the second question even with a modest grid size of 50x50. I was wondering if you had any feedback. I would guess that large beta is the point where the spectral method begins failing but I am not sure.
Shivan Mittal
[hw3.zip](https://github.com/williamgilpin/cphy/files/13172822/hw3.zip)

sm75447

**Reply by shvnmttl** [reply]
[Link](https://github.com/shvnmttl/comphy/blob/90ef109a76e447501f7365a1d33f0d2a423d4c83/hw/hw3.ipynb) to the github version. Note that the huge error log in the file is from keyboard interrupt which I did because of the above-mentioned issue.

**Comment by DarkQuark26** [comment]
https://github.com/DarkQuark26/cphy/blob/main/patrick_devaney_hw_solutions/pde_turing.ipynb

HW 3
Patrick Devaney
UT EID: pd9552

**Comment by ConcreteWang** [comment]
Tong Wang's Homework 3
EID: tw28927
https://github.com/ConcreteWang/Test1/blob/main/pde_turing%20(1).ipynb

**Comment by bluejelli** [comment]
Anish Pandya HW 3
EID ap53535
[hw3](https://gist.github.com/bluejelli/8e28a05db74b53c39c068e217fcfff35)

**Comment by Potatoasad** [comment]
Asad Hussain
[Here's my Homework 3](https://github.com/Potatoasad/cphy/blob/asad_solutions/hw/pde_turing.ipynb)
EID ah54594

**Comment by savpinedo28** [comment]
Savannah Pinedo
sap4423
[https://github.com/savpinedo28/cphy/blob/main/pde_turing_pinedo.ipynb](url)

**Comment by ravikoka** [comment]
Ravi Koka
rkk582
Link to hw4: https://github.com/ravikoka/comp_phy/blob/2a0d774d3b8e50f23babb1402618743577c4da51/hw4_forecasting_regression_supervised.ipynb

**Comment by timothyara** [comment]
Timothy Araujo
tda739
https://github.com/timothyara/test_repo/blob/edf7b9a36c5020f1fedfde643894dcda8f836b7d/TimA_linearregression.ipynb

**Comment by MercuryLYC** [comment]
Yuchen Lin, yl38994
https://github.com/MercuryLYC/test_repo/blob/main/HW4%20yl38994.ipynb

**Comment by abbyperryman** [comment]
Abigail Perryman
asp3337
https://github.com/abbyperryman/cphy/blob/main/hw4.ipynb

**Comment by shoshanareich** [comment]
Shoshi Reich
sar5447
https://github.com/shoshanareich/computational_physics/blob/main/forecasting_regression_supervised.ipynb

**Comment by sepstein22** [comment]
Sophia Epstein
see873
https://github.com/sepstein22/cphy_23/blob/d79b9fb6c0d1cc29659f29593f3829d5fdd18e85/hw/forecasting_regression_supervised.ipynb

**Comment by klanderson42** [comment]
Kody Anderson
kla2648
https://github.com/klanderson42/cphy/blob/hw4/hw/forecasting_regression_supervised.ipynb

**Comment by JImmyHCY** [comment]
Ching-Yu Huang
ch52423
https://github.com/JImmyHCY/CPHY/blob/1a63a1e9cf93d95878e084b674a4e6ccccb6295a/HW4.ipynb

**Comment by bhollo0329** [comment]
Bryce Holloway
bh28929
https://github.com/bhollo0329/CompPhys2023/blob/main/HW4.ipynb

**Comment by laurenlarson** [comment]
Lauren Larson 
lml3638
https://github.com/laurenlarson/cphy/blob/26d324aca4bc93582d52ccaaeb828e73def4e324/hw/HW4.ipynb

**Comment by ccyehintx** [comment]
Chieh-Chih (George) Yeh
cy7658
https://gist.github.com/ccyehintx/22379f21c8ea94ffbda433c2981176be

**Comment by DanUrban000** [comment]
Daniel Urbanski
deu223
https://github.com/DanUrban000/Computational_Physics/blob/main/Urbanski_HW_4.ipynb

**Comment by ConcreteWang** [comment]
Tong Wang
tw28927
https://github.com/ConcreteWang/Test1/blob/main/forecasting_regression_supervised%20(1).ipynb

**Comment by DarkQuark26** [comment]
Patrick Devaney
pd9552
HW 4
https://github.com/DarkQuark26/cphy/blob/main/patrick_devaney_hw_solutions/forecasting_regression_supervised.ipynb

**Comment by ehansen99** [comment]
Erik Hansen, eh29649
[Homework 4](https://github.com/ehansen99/cphy/blob/main/hw/hansen_hw4.ipynb)

**Comment by htphy** [comment]
Han-Ting Chen hc28747
https://github.austin.utexas.edu/hc28747/test_repo/tree/main/hw4

**Comment by nlindsellTexas** [comment]
Noah Lindsell nl22437
https://github.com/nlindsellTexas/CompPhys

**Comment by joseywright** [comment]
https://github.com/joseywright/computationalHomework/blob/main/Homework4.ipynb
Homework 4 jrw5288

**Comment by ycaophysics** [comment]
Yuxuan Cao, yc29393, HW#4
https://gist.github.com/ycaophysics/aaa552bfd40352ee3b36aa982ec690df

**Comment by bluejelli** [comment]
Anish Pandya, ap53535, [HW4](https://gist.github.com/bluejelli/48d56c15e46718ef6e84aaff7d2a47ea)

**Comment by BryanD-UT** [comment]
Bryan Dinh, bvd249 
https://utexas.box.com/s/xbtipyh6airf6vnslmse63nol4gmc8y9

**Comment by gpsehr** [comment]
Gregory Sehr, gps687
https://github.com/gpsehr/CompPhysics/blob/main/Sehr_HW4.ipynb

**Comment by Potatoasad** [comment]
Asad Hussain
EID: ah54594
[Homework 4](https://github.com/Potatoasad/cphy/blob/asad_solutions/hw/forecasting_regression_supervised.ipynb)

**Comment by shvnmttl** [comment]
Shivan Mittal sm75447 [HW4](https://github.com/shvnmttl/comphy/blob/776ae254da6b11c884e7e21e0e0c0437725a7c9c/hw/hw4.ipynb)

**Comment by clobrien-ut** [comment]
Chloe O'Brien
alb5374
https://github.com/clobrien-ut/cphy/blob/main/obrien.forecasting_regression_supervised.ipynb

**Comment by Art-of-FUgue** [comment]
I am sorry I missed the deadlines, I miss read the syllabus. In case it is still acceptable for grading. Any deduction of points is acceptable. 
https://github.com/Art-of-FUgue/HW/blob/main/forecasting_regression_supervised.ipynb
hz6835
Hongming Zhang

**Reply by williamgilpin** [reply]
no worries, have a great thanksgiving!
